src/scattering/eigen_tensor.h

```diff
@@ -0,0 +1,463 @@
+/*
+    pybind11/eigen_tensor.h: Conversion of multi-dimensional numpy arrays to Eigen
+    tensors.
+
+    Copyright (c) 2020 Simon Pfreundschuh <simon.pfreundschuh@chalmers.se>
+
+    All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file.
+*/
+#pragma once
+
+#include "pybind11/pybind11.h"
+#include "pybind11/numpy.h"
```

Comment:
Can you rephrase your concerns please.

Reply:
Yeah, a bit vague.

In short, I don't want `pybind11` in `artscore`.  These includes seem to define ways for you to interact with Eigen tensors using `pybind11` in C++, but it might be that the idea is that you just want these bindings in `pyarts`.  Are you planning to leak this part to `artscore` or have your C++ part depend on `pybind11`?  I would be against that leak.  If you do not plan to leak this to `artscore`, then I would like it if the file is renamed to `py_eigen_tensor.h`

Reply:
The header is only included by ``py_scattering_new.cpp``. So there's no leakage into ``artscore``. It's just an extension of the automatic interfacing capabilities of pybind11 provided by ``arts/scattering``.  Therefore, it makes sense to me to have the file located there, but if you have other preferences I am happy to move it.

Reply:
I see.  Thanks, then I am happy with the file content.  I still want you to name it `py_eigen_tensor.h`, to be clearer with the intent.  It helps to keep the name clean, just as the `m_` names in `artscore` keeps the method files named cleanly.  @olemke , can I get a second opinion on naming policies here?

Reply:
I agree. That's a good convention to follow.